src/electromagnetics.md

```diff
@@ -478,5 +478,85 @@ to specify these regions. To change these, search the code for `std::map<int, do
 integer attribute to the floating-point material value.
 
 
+## Particle Trajectory Calculation
+
+Particle trajectory calculations are an important part of the Particle-In-Cell
+method but they can also provide an illuminating view of electromagnetic fields
+which can otherwise be difficult to visualize.
+
+### Lorentz Mini Application
+
+The Lorentz mini application approximates the trajectory of an individual
+particle using the explicit Boris algorithm. The equations being solved track
```

Comment:
Might be worth citing the original paper here: https://doi.org/10.1063/1.4818428.

Reply:
Done

Reply:
Would not call it the original 😄 as *the* original is from 70s if not older, but it has some nice pictures and explanation 😉 .